milestones/compositeArtifacts.xml

```diff
@@ -2,9 +2,9 @@
 <?compositeArtifactRepository version='1.0.0'?>
 <repository name='TMF Xtext Update Site (Milestones)' type='org.eclipse.equinox.internal.p2.artifact.repository.CompositeArtifactRepository' version='1.0.0'>
   <properties size='1'>
-    <property name='p2.timestamp' value='1539639946675'/>
+    <property name='p2.timestamp' value='1543222972288'/>
   </properties>
   <children size='1'>
-    <child location='head/S201810150206'/>
+    <child location='head/S201811260210'/>
```

Comment:
shall we drop M1?

Reply:
i dont know how this was done in the past.?!? do you remember?

Reply:
am not sure if we catch all errors if p2 can fetch stuff from old update sites

Reply:
Usually the milestones are kept during a release, sometimes also longer. That's why it is a composite and does not hold last MS only.
Effectively noone will use M1 anymore. I'm fine with both.